pkg/auth/metrics.go

```diff
@@ -43,11 +43,12 @@ const (
 )
 
 type Metrics struct {
-	loginRequests        prometheus.Counter
-	loginSuccessful      *prometheus.CounterVec
-	loginFailures        *prometheus.CounterVec
-	logoutRequests       *prometheus.CounterVec
-	tokenRefreshRequests *prometheus.CounterVec
+	loginRequests             prometheus.Counter
+	loginSuccessful           *prometheus.CounterVec
+	loginFailures             *prometheus.CounterVec
+	logoutRequests            *prometheus.CounterVec
+	tokenRefreshRequests      *prometheus.CounterVec
+	internalproxyClientConfig *rest.Config
```

Comment:
Reminder for myself: I need to understand why this is now required and not before.